packages/react-wrapper/tests/demo/stubs/react-user-context.stub.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement, useContext, useState } from 'react'
+import { ReactWrapper } from '../../../src'
+import { VueUserContext } from './vue-user-context'
+import { UserContext } from '../contexts/user-context';
+import styled from 'styled-components'
+
+const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  max-width: 1000px;
+  margin: 0 auto;
+  background-color: beige;
+  padding: 2rem;
+  
+`
```

Comment:
If the dependency from `styled-components` is removed, this would need to be refactored, for example to: 

```
const containerStyle: React.CSSProperties = {
  display: 'flex',
  flexWrap: 'wrap',
  justifyContent: 'space-around',
  maxWidth: '1000px',
  margin: '0 auto',
  backgroundColor: 'beige',
  padding: '2rem'
};
```

And then use it in the render as: 
```
  ...
  return (
    <div style={containerStyle}>
    ...
```